cli/integration_tests/monorepo_one_script_error/run.t

```diff
@@ -64,27 +64,3 @@ Make sure error isn't cached
   
    ERROR  run failed: command  exited (1)
   [1]
-
-Running with --output-mode=errors-only gives error output only
-  $ ${TURBO} --output-logs=errors-only error
-  \xe2\x80\xa2 Packages in scope: my-app (esc)
-  \xe2\x80\xa2 Running error in 1 packages (esc)
-  \xe2\x80\xa2 Remote caching disabled (esc)
-  my-app:error: ERROR: command finished with error: command \(.*/run\.t(-\d+)?/apps/my-app\) npm run error exited \(1\) (re)
-  my-app:error: 
-  my-app:error: > error
-  my-app:error: > echo 'intentionally failing' && exit 2
-  my-app:error: 
-  my-app:error: intentionally failing
-  my-app:error: npm ERR! Lifecycle script `error` failed with error: 
-  my-app:error: npm ERR! Error: command failed 
-  my-app:error: npm ERR!   in workspace: my-app 
-  my-app:error: npm ERR!   at location: .*/run\.t(-\d+)?/apps/my-app  (re)
-  command \(.*/run\.t(-\d+)?/apps/my-app\) npm run error exited \(1\) (re)
-  
-   Tasks:    1 successful, 2 total
-  Cached:    1 cached, 2 total
-    Time:\s*[\.0-9]+m?s  (re)
-  
-   ERROR  run failed: command  exited (1)
-  [1]
```

Comment:
This test was redundant with the new ones I added. The other tests in this file are about caching the outputs, so I kept those.